### PR TITLE
sap_maintain_etc_hosts: Fix wrong assert messages

### PR DESCRIPTION
**Pull request by berndfinger**
Also fixes some typos.

Solves issue #662.

### CODE REVIEW

File: roles/sap_maintain_etc_hosts/tasks/update_host_absent.yml [hunk 1]
```diff
@@ -13,7 +17,7 @@
       ((thishost.node_ip is undefined) and (thishost.node_name is defined))
     msg: "Invalid delete item. Please define either node_ip only or node_name. In the latter case node_domain is optional."
 
-- name: Ensure that the entry all entries in hosts file are removed with IP {{ thishost.node_ip | d('undefined') }}
+- name: Ensure that all entries with IP {{ thishost.node_ip | d('undefined') }} in /etc/hosts are absent
```
Comment (rhmk) [inline roles/sap_maintain_etc_hosts/tasks/update_host_absent.yml:20]: Here and other place did throw a lint error. According to lint variables have to be place a the ended of the name line. That is, why I had  this clumsy expression

File: roles/sap_maintain_etc_hosts/tasks/update_host_absent.yml [hunk 2]
```diff
@@ -26,7 +30,7 @@
   become_user: root
   become: true
 
-- name: Ensure that the entry all entries in hosts file are removed with name {{ thishost.node_name | d('undefined') }}
+- name: Ensure that all entries with name {{ thishost.node_name | d('undefined') }} in /etc/hosts are absent
```
Comment (rhmk) [inline roles/sap_maintain_etc_hosts/tasks/update_host_absent.yml:33]: Here and other place did throw a lint error. According to lint variables have to be place a the ended of the name line. That is, why I had  this clumsy expression

File: roles/sap_maintain_etc_hosts/tasks/update_host_absent.yml [hunk 3]
```diff
@@ -40,7 +44,7 @@
   become_user: root
   become: true
 
-- name: Ensure that the entry all enries in hosts file are removed with FQDN
+- name: Ensure that all enries with the specified FQDN in /etc/hosts are absent
```
Comment (rhmk) [inline roles/sap_maintain_etc_hosts/tasks/update_host_absent.yml:47]: s. above

**Review by rhmk — APPROVED** [review]
In general this looks good. If the commented code lines do not enforce a lint error any more, I am fine with merging

**Comment by berndfinger** [comment]
> In general this looks good. If the commented code lines do not enforce a lint error any more, I am fine with merging
Maybe it's related to the ansible-lint version you have used. The GitHub actions are also clean, so that should be all fine.

**Pull request merged**